libraries/freertos_plus/aws/greengrass/test/aws_test_ggd_system.c

```diff
@@ -84,7 +83,6 @@ TEST_GROUP( GGD_System );
 TEST_SETUP( GGD_System )
 {
     TEST_ASSERT_EQUAL( true, IotSdk_Init() );
```

Comment:
Does IotSdk_Init() returns true mean MQTT Connection success?

Reply:
That function initializes mutexes, static memory, and the taskpool that exist in `libraries/c_sdk/common`